### ISSUE TITLE
scExpr data

### ISSUE THREAD

**Issue by youcef-benmohammed** [humengying0907/deconvBenchmarking#1]
Hi,

In case tpm data is not available, Is it possible to use counts from single cell data to simulate pseudobulk.

Best,
Youcef.

**Comment by humengying0907** [comment]
Yes. But we recommend using library size normalized data for bulk simulation. You can easily convert count data to cpm data using the following code, assuming scExpr is the count data with genes in rows and cells in columns:
```R
col_sums <- Matrix::colSums(scExpr)
scaling_factors <- 10^5 / col_sums
cpm <- scExpr %*% Matrix::Diagonal(x = scaling_factors)
colnames(cpm) = colnames(scExpr)
